os/skia/skia_surface.cpp

```diff
@@ -62,6 +62,13 @@ static SkCanvas::SrcRectConstraint to_constraint(const Paint* paint)
   return SkCanvas::kStrict_SrcRectConstraint;
 }
 
+// static
+Surface::ColorChannelsOrder Surface::getNativeColorChannelsOrder()
+{
+  return kN32_SkColorType == kRGBA_8888_SkColorType ? ColorChannelsOrder::RGB :
+                                                      ColorChannelsOrder::BGR;
```

Comment:
I think these values (`kN32_SkColorType` and `kRGBA_8888_SkColorType`) probably something that can be solved with a `#if` or `if constexpr`?

Reply:
Dont worry, I’ll try something later to have a static check that these constants have the two possible expected values.